internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/token.go

```diff
@@ -157,5 +188,67 @@ func (b *Bridge) tokenForSink(ctx context.Context, now time.Time, sink definitio
 	)
 
 	b.logger.Info(ctx, "created token")
-	return newToken, nil
+	return *result, nil
+}
+
+func (b *Bridge) MigrateTokens(ctx context.Context) error {
```

Comment:
I think you have issue, that this migration is run with each restart of program. Can you verify it in test? Then add some protection that migration can be done only once. That's why I would rather do this function inside command line

Reply:
Also can we try it saving into another prefix just for testing ? I see the test but be rather sure that we do not override the tokens incorrectly